src/main/webapp/pages/projects/settings/pages/processing.html

```diff
@@ -12,145 +12,146 @@
         }
```

Comment:
There is a bunch of html that is commented out in this file. If it is not required please remove it :)

Reply:
Thanks, I really meant to do that.  Updated in f4df94e